lib/engine/game/g_18_uruguay/nationalization.rb

```diff
@@ -28,10 +28,13 @@ def acquire_shares
             entity_shares = affected_shares(holder, @merge_data[:corps])
 
             total_percent = entity_shares.sum(&:percent)
-            num_shares, odd_shares = (total_percent / 10).divmod(2)
-            from_secondary = @merge_data[:secondary_corps].count { |corp| corp.president?(holder) }
+            num_shares = (total_percent / 20).to_i
```

Comment:
@patrikolesen Did you mean to revert this change in this commit? What was rubocop objecting to here? Or was this commit just supposed to remove the space after `bundle =` below?